development/README.md

```diff
@@ -286,167 +286,309 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
+
+  ```
+  HOMEBREW_PREFIX=/usr/local
+  PATH=/usr/local/opt/ccache/libexec:/usr/local/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin
+  ```
 
+  **Notes:**
+  * The kit's Qt5 `bin` dir is always prepended to PATH: `/usr/local/opt/qt5/bin`, but it will be resolved to it's "Cellar" path, e.g. `/usr/local/Cellar/qt5/5.7.1_1/bin`, which is not an issue since it is dynamically prepended.
+  * `HOMEBREW_PREFIX` is used by the custom build and install scripts.
 
-* **Debugger:** System LLDB at /Library/Developer/CommandLineTools/usr/bin/lldb
-* **Qt Version:** Qt 5.7.0 (local)
+* **Debugger:** System LLDB at `/Applications/Xcode.app/Contents/Developer/usr/bin/lldb` (or `/usr/bin/lldb`, if using the Command Line Tools)
+* **Qt Version:** Brew Qt5 (which you previously created)
 * **Qt mkspec:** leave blank
-* **CMake Tool:** Brew CMake (which you should have created further up in these notes)
-* **CMake Generator:** CodeBlocks - Unix Makefiles (ninja is not working for me)
-* **CMake Configuration:** Press the ‘Change…’ button and paste this into the box provided:
-
-
-```
-CMAKE_CXX_COMPILER:STRING=%{Compiler:Executable}
-QT_QMAKE_EXECUTABLE:STRING=%{Qt:qmakeExecutable}
-CMAKE_INSTALL_PREFIX:PATH='/Users/timlinux/Applications/' 
-CMAKE_FIND_FRAMEWORK:STRING=LAST 
-CMAKE_PREFIX_PATH:STRING='/usr/local/opt/qt5;/usr/local/opt/qt5-webkit;/usr/local/opt/gdal2;/usr/local/opt/expat;/usr/local/opt/sqlite;/usr/local/opt/flex;/usr/local/opt/bison' 
-ENABLE_MODELTEST:BOOL=FALSE 
-ENABLE_TESTS:BOOL=TRUE 
-GDAL_LIBRARY:FILEPATH=/usr/local/opt/gdal2/lib/libgdal.dylib 
-GEOS_LIBRARY:FILEPATH=/usr/local/opt/geos/lib/libgeos_c.dylib 
-GSL_CONFIG:FILEPATH=/usr/local/opt/gsl/bin/gsl-config 
-GSL_INCLUDE_DIR:PATH=/usr/local/opt/gsl/include 
-GSL_LIBRARIES:STRING='-L/usr/local/opt/gsl/lib -lgsl -lgslcblas' 
-WITH_QWTPOLAR:BOOL=TRUE 
-WITH_INTERNAL_QWTPOLAR:BOOL=FALSE 
-WITH_GRASS:BOOL=FALSE 
-WITH_GRASS7:BOOL=TRUE 
-GRASS_PREFIX7:PATH=/usr/local/opt/grass7/grass-base 
-WITH_APIDOC:BOOL=FALSE 
-WITH_ASTYLE:BOOL=TRUE 
-WITH_CUSTOM_WIDGETS:BOOL=TRUE 
-WITH_GLOBE:BOOL=FALSE 
-WITH_ORACLE:BOOL=FALSE 
-WITH_QSCIAPI:BOOL=FALSE 
-WITH_QSPATIALITE:BOOL=FALSE 
-WITH_QTWEBKIT:BOOL=TRUE 
-WITH_SERVER:BOOL=TRUE 
-WITH_STAGED_PLUGINS:BOOL=TRUE 
-QGIS_MACAPP_BUNDLE:STRING=0
-```
-
-**Note:** Most of these settings were taken from $(brew --repository qgis/qgisdev)/scripts/qgis-cmake-setup.sh. You can comment out the last line of the aforementioned script and then just echo the CMD instead of running it to get the equivalent options above.
+* **CMake Tool:** Brew CMake (which you previously created)
+* **CMake Generator:** CodeBlocks - Unix Makefiles
+* **CMake Configuration:** Press the `Change...` button and paste this into the box provided:
 
+  ```
+  CMAKE_CXX_COMPILER:STRING=%{Compiler:Executable}
+  CMAKE_C_COMPILER:STRING=%{Compiler:Executable:C}
+  QT_QMAKE_EXECUTABLE:STRING=%{Qt:qmakeExecutable}
+  CMAKE_BUILD_TYPE:STRING=RelWithDebInfo
```

Comment:
Yes. I think I added this prior to realizing that the Configure Project panel offered the selection of build configurations _prior_ to configuring the project (I think it used to just run default config first).